maquetado/css/styles.css

```diff
@@ -34,3 +33,35 @@
     display: none;
   }
 }
+
+.features {
+  display: flex;
+  margin-top: 100px;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+b {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.bold {
+  margin-top: 20px;
```

Comment:
it's preferred to use margin bottom instead of top, you should add a class to the images and add them a margin bottom and add a margin bottom here to separate the title from the paragraph.